VisualStudio.gitignore

```diff
@@ -250,3 +250,8 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+
+# NCrunch
+_NCrunch_SOLUTION/*
```

Comment:
Is `SOLUTION` a placeholder here? I'm puzzled as to why we've not added this earlier...